### PR TITLE
Dev user info impl

### PR DESCRIPTION
**Pull request by leechristophermurray**
profile.html was updated to retrieve info from "current_user" rather than the now secure "user" model/endpoint.

This pull request is mainly to update the view so it is compliant with how the endpoints are now secure. The changes prior made the fields return empty. I rectified the matter before an issue had been created.

### CODE REVIEW

File: routes/user/user.js [hunk 1]
```diff
@@ -247,3 +247,19 @@ exports.activateUser = function(req, res, next) {
             }
         });
 };
+
+exports.updateUser = function(req, res, next) {
```
Comment (matjames007) [inline routes/user/user.js]: @thinkking What is the endpoint used to access this function?  Also we shouldn't be retrieving using the token.  Try finding by user id.  I suggest: `PUT /user/:id`

In HTTP web services PUT is seen as an update to the resource.

File: public/javascript/controllers/user-ctrl.js [hunk 2]
```diff
@@ -70,6 +70,20 @@ angular.module('harvestv2')
...
                 }
             };
+            $scope.chpass = function () {
+                if (!$scope.chpassForm.$invalid &&
+                    $scope.validatePasswordMatch($scope.user.us_password, $scope.passwordConfirm) &&
+                    $scope.agreeChecked) {
+
+                    $scope.user.us_password = CryptoJS.SHA1($scope.user.us_password).toString(CryptoJS.enc.Hex);
+
+                    UserFactory.create($scope.user, function(user) {
```
Comment (matjames007) [inline public/javascript/controllers/user-ctrl.js]: yikes... @thinkking we might have to create a separate endpoint for password change.  `UserFactory.create` will be treated as a registration and should complain that the user already exists

Comment (leechristophermurray) [reply public/javascript/controllers/user-ctrl.js]: Okay, will do.

File: models/db.js [hunk 3]
```diff
@@ -6,7 +6,7 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
-mongoose.connect(process.env.MONGOLAB_URI);
+mongoose.connect(process.env.MONGOLAB_URI || "mongodb://localhost:27017/mongodb");
```
Comment (matjames007) [inline models/db.js]: @thinkking remember to remove this form your commits.

**Comment by paradoxxjm** [comment]
@thinkking could you update this to include which issue number that this addresses.

**Comment by matjames007** [comment]
Was this supposed to include the ability to change the password from the user profile page?

**Comment by leechristophermurray** [comment]
@paradoxxjm and @matjames007 The pull request description has been changed to clarify its purpose.

**Review by matjames007 — APPROVED** [review]
Next time please tag the associated issue that this rectifies.  You can refer to the issue by doing `#<issue_number>`.

**Pull request merged**